src/components/GamePage/GamesSubmenu.tsx

```diff
@@ -79,6 +86,21 @@ export default function GamesSubmenu({
     return
   }
 
+  async function handleUpdate() {
+    const { response } = await showMessageBox({
+      title: t('box.update.title'),
+      message: t('box.update.message'),
+      buttons: [t('box.yes'), t('box.no')],
+    })
```

Comment:
Do we want to have the fallback language at every translation?

Reply:
The fallback language will always be english, you mean on every key a fallback string? I'm doing this only with new string because then the parser adds the default to all json files and then it is easier to add the translations afterwards.

Reply:
Alrighty! :)